### ISSUE TITLE
Return value of Game::worldPosToZonePos()

### ISSUE THREAD

**Issue by DerEnderman** [Sarjuuk/aowow#209]
```23:23:48  [OK]     - subscript 'achievementcriteria' returned sucessfully
23:23:48          SqlGen::generate() - filling aowow_areatrigger with data
23:23:49             - fetching taverns
23:24:27             - calculation teleporter coordinates
Exception - Return value of Game::worldPosToZonePos() must be of the type array, boolean returned
   /var/www/vhosts/sdsdsd.de/httpdocs/aowow/includes/game.php(267)
#0 /var/www/vhosts/sdsdsd.de/httpdocs/aowow/setup/tools/sqlgen/areatrigger.func.php(43): Game::worldPosToZonePos(189, 1053.48, 1688.99)
#1 /var/www/vhosts/sdsdsd.de/httpdocs/aowow/setup/tools/sqlGen.class.php(205): class@anonymous->generate(Array)
#2 /var/www/vhosts/sdsdsd.de/httpdocs/aowow/setup/tools/clisetup/sql.func.php(33): SqlGen::generate('areatrigger', Array)
#3 /var/www/vhosts/sdsdsd.de/httpdocs/aowow/setup/setup.php(71): sql()
#4 /var/www/vhosts/sdsdsd.de/httpdocs/aowow/aowow(10): require('/var/www/vhosts...')
```

Can you investigate the issue?

**Comment by Sarjuuk** [comment]
the contained query has to fail for that function not to return an array.

would you mind modifying the following line and try again:
https://github.com/Sarjuuk/aowow/blob/master/setup/tools/sqlgen/areatrigger.func.php#L15  
```diff
-    protected $dbcSourceFiles  = ['areatrigger'];
+    protected $dbcSourceFiles  = ['areatrigger', 'worldmaparea', 'dungeonmap'];
```

**Comment by DerEnderman** [comment]
i will try and recheck the result. i will edit this post with news to you

**Comment by DerEnderman** [comment]
![](https://i.imgur.com/VvydAwm.png)
Hmmm something anoter issues ? and 

```
continue setup? (y/n):
20:16:29          Resuming setup from step 2...
20:16:30  [ERR]   SqlGen::generate - invalid table given

[c]ontinue anyway? [r]etry? [a]bort?:
20:16:32  [INFO]  generated dbc_* - tables kept available
```

Seems dbc has been changed?

**Comment by Sarjuuk** [comment]
should be fixed by 844a4c0e527a16377c9627ea1c89129baeb1cf68

the warnings are expected unless you manually add data for instanced areas

**Comment by Sarjuuk** [comment]
assumed working
